servicetalk-dependencies/build.gradle

```diff
@@ -23,8 +23,17 @@ javaPlatform {
   allowDependencies()
 }
 
+// Include modules directly rather than by defining servicetalk bom as
+// a platform to work around [MNG-6772](https://issues.apache.org/jira/browse/MNG-6772).
+rootProject.subprojects.findAll { !it.name.endsWith("-bom") &&
```

Comment:
Consider adding a comment that this part have to be consistent with what we have in `servicetalk-bom` and vice versa.

Reply:
Comment added in both files.